rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -128,6 +128,22 @@
             @QueryParam("index") @DefaultValue("-1") int index, @QueryParam("limit") @DefaultValue("-1") int limit)
             throws PermissionRestException, NotConnectedRestException;
 
+    /**
+     * Returns a list of jobs with each jobs is described using -
+     * its id - its owner - the JobInfo class
+     *
+     * @param jobsId
+     *            the list of id of the jobs to return
+     * @param sessionId
+     *            a valid session id
+     * @return a list of UserJobData
+     */
+    @GET
+    @Path("listjobinfo")
```

Comment:
I would have rather named the endpoint `joblistinfo`, but we also need another opinion on that.

Reply:
Unfortunately we have already "jobsinfo". To differenciate, we can call the enpoint (and methods) jobsinfolist

Reply:
I agree that the best would be to rename `jobsinfo` into `alljobsinfo` and to name this one `jobsinfo`, but since we can't afford to do that now, I agree with `jobsinfolist`.